common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -66,6 +66,23 @@
     @Supported
     public static final KnownKey zimbra_tmp_directory = KnownKey.newKey("${zimbra_home}/data/tmp");
 
+    /**
+     * Sets whether the External Email Warning (EEW) feature is enabled.
+     */
+    @Supported
+    public static final KnownKey zimbra_eew_is_enabled = KnownKey.newKey(false);
+
+    /**
+     * Sets the base admonition or warning for External Email Warning (EEW).
+     * 
+     * This string will be used and adapted for both text/plain and text/html parts
+     * in a message. Please do not attempt to use HTML mark up here, it will be
+     * escaped anyways.
+     */
+    @Supported
+    public static final KnownKey zimbra_eew_base_admonition = KnownKey
```

Comment:
for this attribute `zimbra_external_email_warning_message` would be better

Reply:
Done.